README.md

```diff
@@ -80,36 +79,18 @@ Once an application is 'working' it is deployed in dev, test, and prod environme
  * may be 'golden' vm on Developer desktop or vm provided by Operations for this purpose
  * no backups!
 
-**Using [Vagrant](Vagrant/README.md) to create a development environment on your workstation is recommended.**
-
-####Test Environment
+####User Acceptance Testing (UAT) Environment
```

Comment:
Seems to be some markdown issues here, none of these are rendering as headers? Probably need a space after the hashtags i assume?